### PR TITLE
Make a[role=button] consistent with button styling

### PR DESCRIPTION
**Pull request by pietvanzoen**

### CODE REVIEW

**Comment by pietvanzoen** [comment]
Somewhat related... is there a particular reason for these two sections being separate?

https://github.com/pietvanzoen/simple.css/blob/8a6271fdb3cb830537d77732de696b041a160c0e/simple.css#L205-L225

File: simple.css [hunk 1]
```diff
@@ -180,6 +180,7 @@ a:hover {
 
 button,
 [role="button"],
+a[role="button"],
```
Comment (lkhrs) [inline simple.css:183]: we're already selecting for `[role="button"]`, do we need a duplicate selector for `a`?

**Review by lkhrs — CHANGES_REQUESTED** [review]

**Comment by lkhrs** [comment]
> Somewhat related... is there a particular reason for these two sections being separate?
> 
> https://github.com/pietvanzoen/simple.css/blob/8a6271fdb3cb830537d77732de696b041a160c0e/simple.css#L205-L225

No, that could be cleaned up. We do need to avoid giving form inputs the same disabled style as buttons though.

**Comment by lkhrs** [comment]
What is your use case for anchor tags as buttons? Would the `<button>` element solve your needs?

I know `role="button"` was already in the stylesheet, but I'm wondering if we should remove styling for it entirely. Elements using that role need extra work to make accessible (such as listening for key events).

File: simple.css (hunk 1 above)
Comment (pietvanzoen) [reply simple.css:183]: Good point!

**Comment by pietvanzoen** [comment]
> What is your use case for anchor tags as buttons? Would the `<button>` element solve your needs?
> 
> I know `role="button"` was already in the stylesheet, but I'm wondering if we should remove styling for it entirely. Elements using that role need extra work to make accessible (such as listening for key events).

My use case is that I’d like a row of anchors and buttons to visually look the same. 

The anchors are still regular anchors that link to other resources.

**Comment by lkhrs** [comment]
In that case, the `.button` class would be a more accessible approach. Looks like it needs `line-height: 1` to make it consistent. We should combine it with the styling for form buttons in a separate PR, and remove styling for `role="button"` in favor of the `.button` class—is that something you'd like to tackle?

I'd rather not provide styling for `role="button"` due to [the JavaScript needed](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#accessibility_concerns) to make those elements accessible, and encourage use of the `.button` class by default.

**Pull request closed without merging**